core/math/vector3.h

```diff
@@ -214,7 +214,7 @@ real_t Vector3::distance_squared_to(const Vector3 &p_b) const {
 
 real_t Vector3::angle_to(const Vector3 &p_b) const {
 
-	return Math::acos(this->dot(p_b) / Math::sqrt(this->length_squared() * p_b.length_squared()));
+	return Math::atan2(cross(p_b).length(), dot(p_b));
```

Comment:
Why this formula and not the "method 2" mentioned [on this Math SE](http://math.stackexchange.com/questions/1143354/numerically-stable-method-for-angle-between-3d-vectors/1782769)?

I.e. why not `2 * Math::atan2((normalized() - p_b.normalized()).length(), (normalized() + p_b.normalized()).length())`